### PR TITLE
Hotfix/swas/sdk iframe removal

### PR DESCRIPTION
**Pull request by swashata**

### CODE REVIEW

File: includes/class-freemius.php [hunk 1]
```diff
@@ -1380,16 +1380,35 @@ function _run_garbage_collector() {
...
         /**
-         * Opens the support forum subemenu item in a new browser page.
+         * Modifies all external links in the submenu by altering their href, and also opens them in new tab if needed.
          *
          * @author Vova Feldman (@svovaf)
+         * @author Swashata Ghosh (@swashata)
          * @since  2.1.4
          */
-        static function _open_support_forum_in_new_page() {
+        static function _handle_submenu_external_link() {
```
Comment (swashata) [inline includes/class-freemius.php:1389]: This now handles external links and opening new tab for any SDK introduced submenu link. @fajardoleo please also help test with tabs.

File: includes/managers/class-fs-checkout-manager.php
Comment (swashata) [inline includes/managers/class-fs-checkout-manager.php:1]: Code extracted from the `templates/checkout.php` file.

File: includes/managers/class-fs-contact-form-manager.php
Comment (swashata) [inline includes/managers/class-fs-contact-form-manager.php:1]: Code extracted from `templates/contact.php` file.

File: templates/checkout.php
Comment (swashata) [inline templates/checkout.php:1]: The GitHub UI is showing this, but I did it in a way where the `templates/checkout.php` was first moved to `templates/checkout/frame.php` and then I did further changes. So taking a look at the blame history, I can see it is there.

File: templates/checkout/frame.php
Comment (swashata) [inline templates/checkout/frame.php:1]: Please check your PHPStorm's git annotation to make sure the git history is intact and it is only GitHub UI, not showing the file rename:
CleanShot 2024-09-30 at 16 13 10@2x

File: templates/checkout/redirect.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+				),
+				$fs->get_unique_affix() . '_activate_new'
+			),
+		)
+	);
+	$query_params['install_data'] = json_encode( $install_data );
+
+	$query_params['_fs_dashboard_independent'] = true;
+
+	fs_redirect( $fs_checkout->get_full_checkout_url( $query_params ) );
```
Comment (swashata) [inline templates/checkout/redirect.php:73]: Please confirm that using `fs_redirect` inside template is something we can do. I didn't find any issue though.

**Review by swashata — COMMENTED** [review]
@fajardoleo please see my comments, thanks.

**Review by fajardoleo — APPROVED** [review]
LGTM

**Pull request merged**